docs/io/ascii/fixed_width_gallery.rst

```diff
@@ -33,8 +33,8 @@ Reading
   EXAMPLE START
   Reading Fixed-Width Tables
 
-FixedWidth
-----------
+Fixed Width
+-----------
```

Comment:
I'm not going to argue about this - the old form is as good as a new one - but if we make the step from "Python-Like Class name" as caption to "real text", should this not be "Fixed width", "Fixed width without header"?
I can see an argument for keeping it in the middle, as well, just commenting.

Reply:
I think that section titles are always capitalized like that.

Reply:
OK, that's fine, too.